src/org/apache/commons/httpclient/HttpMethodDirector.java

```diff
@@ -410,7 +410,7 @@ private void applyConnectionParams(final HttpMethod method) throws IOException {
             param = this.conn.getParams().getParameter(HttpConnectionParams.SO_TIMEOUT);
         }
         if (param != null) {
-            timeout = ((Integer)param).intValue();
+            timeout = (Integer) param;
```

Comment:
Would you mind reverting this change? (Be good to keep the changes to these forked classes to a minimum necessary for our use cases.)

Reply:
@thc202,
You are right. I'll revert this one.